proposals/nnnn-structured-concurrency.md

```diff
@@ -358,9 +358,20 @@ async let result = try fetchHTTPContent(of: url)
 
 Any reference to a variable that was declared in an `async let` is a suspension point, equivalent to a call to an asynchronous function, so it must occur within an `await` expression. The initializer of the `async let` is considered to be enclosed by an implicit `await` expression.
 
-If the initializer of the `async let` can throw an error, then each reference to a variable declared within that `async let` is considered to throw an error, and therefore must also be enclosed in one of `try`/`try!`/`try?`. 
+If the initializer of the `async let` can throw an error, then each reference to a variable declared within that `async let` is considered to throw an error, and therefore must also be enclosed in one of `try`/`try!`/`try?`:
```

Comment:
Do we want to say "within that `async let` clause" to hint at the case where there are multiple, comma-separated clauses within the same `async let`?

Reply:
Sounds good! I can also spell out the closure in one of the examples I suppose!